trustmanager/filestore_test.go

```diff
@@ -416,3 +333,47 @@ func generateRandomFile(filePath string, perms os.FileMode) ([]byte, error) {
 
 	return rndBytes, nil
 }
+
+func TestFileStoreConsistency(t *testing.T) {
+	// Temporary directory where test files will be created
+	tempBaseDir, err := ioutil.TempDir("", "notary-test-")
+	assert.NoError(t, err)
+	defer os.RemoveAll(tempBaseDir)
+
+	s, err := NewPrivateSimpleFileStore(tempBaseDir, "txt")
```

Comment:
Maybe also create a `NewPrivateSimpleFileStore(tempBaseDir, ".txt")` to also test that behavior?